### ISSUE TITLE
Reboot button is not working

### ISSUE THREAD

**Issue by JamesMoore2090** [FalconChristmas/fpp#1894]
FPP Version:	8.x-master-371-g1fe9c75e
Platform:	Raspberry Pi (Pi 4)
Local Git Version: | 1fe9c75

Clicking the reboot button doesn't work. I have a ssh session open as well. I expect the ssh session to be bumped when the system is rebooted. There is no second action to confirm like there is on the BBB version.

**Comment by cybercop23** [comment]
IDK.. try latest udpate.. works for me....
![image](https://github.com/user-attachments/assets/a46775ab-93f5-4094-9514-0203c400a508)

**Comment by JamesMoore2090** [comment]
I am going to re-image the sd card. I'll let you know how it goes later tonight.

**Comment by cybercop23** [comment]
PLEASE don't reimage. just upgrade. there's no OS changes.. simple upgrade.
If you need to, you can always just install FPPOS to replace all.

**Comment by JamesMoore2090** [comment]
I did not reimage. I upgraded the app version to the latest from GitHub. 
<img width="1470" alt="Screenshot 2024-07-27 at 7 21 00 PM" src="https://github.com/user-attachments/assets/adbc4261-3291-459f-a868-e473e584b93d">

I also physically removed power and rebooted. Still no joy. 

I build out a spare SD card and started up a new PI4. Went through the initial setup. The reboot button worked. I also backed up the original issue pi and then restored it to the new. The reboot button again worked.
